internal/util/update/update.go

```diff
@@ -105,6 +104,9 @@ type Command struct {
 	// Path is the filepath to the local package
 	Path string
```

Comment:
Should we rename this path to `RelPath` to be explicit?

Reply:
I want to remove this property, but I want to do in a couple of steps to avoid a large refactor.